trunk/NDHMS/hydro_namelists.json

```diff
@@ -31,6 +31,10 @@
             "output_gw": 1,
             "ovrtswcrt": 1,
             "reservoir_persistence": false,
+            "reservoir_persistence_parameter_file": "NULL_specified_in_domain.json",
```

Comment:
These lines in particular are now what's causing a test failure in CI.  When removed all tests pass locally.  

I'm guessing these are not yet specified in the testing domains for CI and that's the issue.  If we're not interested in actively testing this persistence code (for now) we should probably just remove these, but I haven't been involved in the testing discussions surrounding this work so I'll let someone else chime in.

Reply:
As mentioned in the meeting on 9/26, let's take these out for now so the testing passes. Then add them back in a new PR (with the appropriate files added).

Reply:
I just wasn't sure if something else was going on on the domain side.  I'll push that commit.

Reply:
I think the tests should not be failing with these lines in as long as reservoir_persistence is set to false. You can paste the outputs if you want me to look into this.

I have a couple small updates to commit to only handle some variable name changes in that parameter file. Should I add them to this PR or wait until it is merged and add them to the next PR?

Reply:
I'm guessing the candidate run was failing because the domain json file for Croton (domain used for CI) hadn't yet been updated to include these.  In this case it failed the test (run candidate) without an error reported.

Reply:
If the updates are to code, I'd just go ahead and add them now.  It they're to the missing reservoir options in the hydro_namelist.json file let's hold off until the domain files, etc. get added.

Reply:
Okay I think that makes sense. In our testing, the code ignores these parameters when reservoir_persistence is set to false. The NULL_specified_in_domain.json might be requiring them in the domain regardless of whether it is set to true or false.

I will push those updates now, which are just the code and not the hydro_namelist.json. That reservoir parameter file will also be in the Reservoirs/Testing_Files directory.

Reply:
Yeah, the failure wasn't in the actual model run (code) rather the testing so that checks out.    

Sounds good on the code updates, but I don't think we want a parameter file in the repo.  Those are stored externally.